app/src/main/res/values/preferences.xml

```diff
@@ -123,6 +123,12 @@
     <string name="preference_title_insistent_alarms_enabled">Enable insistent alarm</string>
     <string name="preference_summary_insistent_alarms_enabled">Keeps an alarm on infinitely.</string>
 
+    <!-- UI -->
```

Comment:
Please replace `UI` with `User interface`. Its easier to search for and provides more context for new people joining the project.